pyproject.toml

```diff
@@ -14,6 +14,7 @@ azure-identity = "^1.12.0"
 azure-mgmt-datafactory = "^3.1.0"
 azure-storage-file-datalake = "^12.11.0"
 polling2 = "^0.5.0"
+great-expectations = "^0.15.15"
```

Comment:
Why do we need GX in the top level pyproject.toml?

Reply:
We don't, I got in a battle with the lock files and it's clearly ended up in the wrong place I'll remove it